spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/AboutControllerTests.java

```diff
@@ -49,8 +48,7 @@
 /**
  * @author Glenn Renfro
  */
-@ExtendWith(SpringExtension.class)
-@SpringBootTest(classes = TestDependencies.class)
+@SpringBootTest(classes = {TestDependencies.class, ProjectInfoAutoConfiguration.class, InfoContributorAutoConfiguration.class},properties = {"management.info.git.enabled=true"})
```

Comment:
I did not look at this test close enough earlier, but I think lets remove the info auto configs from the top-level SBT and instead focus on these new features in the `AboutTests` at the very bottom.

Reply:
Done!